R/export_transect_evdata.R

```diff
@@ -33,7 +33,7 @@ export_transect_evdata <- function(prjdir, transectname, horizbin) {
   EVFile<-EVAppObj$OpenFile(EVFile2Open)
 
   # Export Cruise Track
-  GPS<-EVFile[["Variables"]]$FindByName("Fileset1: position GPS fixes")
+  GPS<-EVFile[["Variables"]]$FindByName("Fileset1: position GPS fixes (1)")
```

Comment:
This section will require an if statement to check for one, then, if not found, check for the other line.

Reply:
opened issue #7 to resolve code change requirements